model/event/subscriptions_test.go

```diff
@@ -223,7 +226,7 @@ func (s *subscriptionsSuite) TestRegexSelectorsMatch() {
 }
 
 func (s *subscriptionsSuite) TestFindByOwner() {
```

Comment:
Can you change the name of this to TestFindByOwnerForPerson(), then add another test TestFindByOwnerForProject()?

Reply:
added